### PR TITLE
[v2.0.6] Study builder, Issues fixed 3685 3653

### PR DESCRIPTION
**Pull request by harisboston**
Fixed issues :
[SB] Notifications > Change the error message text when entered time has expired #3685

[SB] Text in the pop-up should be center-aligned #3653

### CODE REVIEW

File: study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/notification/createOrUpdateNotification.jsp [hunk 1]
```diff
@@ -438,7 +438,7 @@
         $('#timepicker1').val('');
         $('.timepicker').parent().addClass('has-error has-danger').find('.help-block.with-errors')
             .empty().append(
-            	$("<ul><li> </li></ul>").attr("class","list-unstyled").text("Please select a time that has not already passed for the current date"));
+            	$("<ul><li> </li></ul>").attr("class","list-unstyled").text("Please select a time that has not gone past yet"));
```
Comment (moschetti) [inline study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/notification/createOrUpdateNotification.jsp:441]: This wording sounds incorrect. Please use "Please select a time in the future."

File: study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/addOrEditStudyNotification.jsp [hunk 2]
```diff
@@ -473,7 +473,7 @@ For studies that are already launched, notifications get scheduled for delivery
       if (dt < serverDateTime()) {
         $('.timepicker').parent().addClass('has-error has-danger').find(
             '.help-block.with-errors').empty().append($("<ul><li> </li></ul>").attr("class","list-unstyled").text(
-            "Please select a time that has not already passed for the current date"));
+            "Please select a time that has not gone past yet"));
```
Comment (moschetti) [inline study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/addOrEditStudyNotification.jsp:476]: This wording sounds incorrect. Please use "Please select a time in the future."

**Review by moschetti — CHANGES_REQUESTED** [review]

File: study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/notification/createOrUpdateNotification.jsp (hunk 1 above)
Comment (harisboston) [reply study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/notification/createOrUpdateNotification.jsp]: Fixed review comment

File: study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/addOrEditStudyNotification.jsp (hunk 2 above)
Comment (harisboston) [reply study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/addOrEditStudyNotification.jsp]: Fixed review comment

**Review by moschetti — APPROVED** [review]

**Pull request merged**